### PR TITLE
Reorganize the OCABundles folder and update README.md

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Comment by swcurran** [comment]
This PR addresses the problem with the repo not being able to be cloned on Windows by reorganizing the OCABundles folder to use "human" folder names, and to updates the process for using the repo by generating a JSON file that provides the OCA Bundles by identifier.  This addresses issue #22.

@amanji -- assuming this gets merged, can you please update the OCA Explorer dropdown to use the `ocabundleslist.json` for loading the OCA Bundles?  Or at least update the existing mechanism to use the new path to the OCA Bundles.

@jcdrouin21 -- please take a look at this PR and make sure this works for you.  Note that I didn't change the paths to the images in your bundle.json file (and OCA Bundle) as I didn't know what to set them to.  Happy to make that change if it would help.  As you will see, they are pointing to what will be the wrong place in your fork of the repo once (if) both this PR is merged and you update your fork.

FYI: @esune

**Comment by swcurran** [comment]
@amanji — looks like your try at a fix didn’t get signed.  Can you fix, or do I need to recreate the PR — which would not be fun :-)

Also need an approval after that is done.  We can fix that typo after.

**Review by amanji — APPROVED** [review]

**Pull request merged**
